src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -261,15 +261,7 @@ private AddTxResult AddCore(Transaction tx, bool isPersistentBroadcast, bool isR
 
             Metrics.PendingTransactionsReceived++;
             
-            IReleaseSpec releaseSpec = _specProvider.GetSpec();
-            if (tx.Type == TxType.AccessList && !releaseSpec.IsEip2930Enabled)
```

Comment:
ok, previously types like 5, 6, 7 would not throw exception and be allowed by the system